lib/components/UI/base-styles.css

```diff
@@ -3,3 +3,6 @@
 
 .pull-left {
   float: left; }
+
+.link {
+  cursor: pointer; }
```

Comment:
^ I didn't add this, so I'll be honest, I have no idea where it came from. Nor do I understand why we'd be using anything other than an anchor tag for links. 🤔

Reply:
Not sure either, maybe it is a button as a link?

Reply:
Yeah we shouldn't be doing that ever though! 

Any idea how this might have been able to end up in my PR, if I didn't add it? :/